aips/aip-x-safe-auth-key-mapping.md

```diff
@@ -0,0 +1,143 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Safe onchain key rotation address mapping for standard accounts
+author: Alex Kahn (43892045+alnoki@users.noreply.github.com)
+discussions-to (*optional): https://github.com/aptos-foundation/AIPs/issues/487
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard Framework
+created: 09/17/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Safe onchain key rotation address mapping for standard accounts
+
+## Summary
+
+The onchain key rotation address mapping has functional issues which inhibit
+safe mapping of authentication key to originating address for standard accounts.
+This proposal resolves these issues by adding assorted checks and extra function logic.
+
+### Out of Scope
```

Comment:
> This feels out of place

@davidiw This section is included because of your previous inquiry about how how the reference implementation will affect multisig v2, which I answered in the AIP issue that I was first instructed to file, under the section "Multisig considerations": https://github.com/aptos-foundation/AIPs/issues/487

I also included it here so that this PR matches the issue

Do you want me to remove it?